### DISCUSSION TITLE
Receiving context.author object outdated

### DISCUSSION THREAD

**Discussion by iHazzu** [Rapptz/discord.py#6408] [Q&A] [answered]
Prior to version 1.6 I reported receiving the outdated member object using `guild.fetch_member` (https://github.com/Rapptz/discord.py/discussions/6034). This was resolved in the new version. However, I still receive the outdated author of the messages. For example, when the member changes the avatar after connecting the bot to the discord, the link to the avatar is not updated when he requests a new command. Its a bug or i need to enable members intents to get updated `context.author` object?
Here is the bot object:
```python
bot = commands.Bot(
    command_prefix = db.prefix,
    case_insensitive=True,
    help_command=None,
    intents=Intents(messages=True, reactions=True, guilds=True, emojis=True),
    member_cache_flags=MemberCacheFlags.none(),
    max_messages=None
)
```

**Comment by Rapptz** [comment, marked as answer]
This shouldn't be possible unless Discord is sending stale data. With the member intent disabled or member cache disabled all user data is no longer cached by the library at all which means that it's entirely pass-through from what Discord gives.

**Reply by iHazzu** [reply]
Oh, sorry. After your answer I looked more carefully at the bot commands and saw that I used `guild.query_members` somewhere and forgot to set `cache = False`. This caused the bot to store stale members
